packages/service-library/package.json

```diff
@@ -39,6 +37,8 @@
         "typescript": "^3.8.3"
     },
     "dependencies": {
+        "@azure/cosmos": "^3.6.2",
```

Comment:
is there a reason to move it from dev-dependencies to dependencies?

Reply:
because these are not dev dependencies. (dev dependencies are the dependencies you use at compile time. like typings, webpack etc.